src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        String line;
+        HashMap<String, Integer> st = new HashMap<>();
+        try (BufferedReader bri = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter bro = new BufferedWriter(new FileWriter(toFileName))) {
```

Comment:
“write to file” or “create report” - are a separate operations. So you should create separate methods for them.